components/brave_wallet_ui/utils/string-utils.ts

```diff
@@ -121,3 +121,13 @@ export const endsWithAny = (extensions: string[], url: string) => {
     return url.endsWith(suffix)
   })
 }
+
+export const getNFTTokenStandard = (token: BraveWallet.BlockchainToken) => {
```

Comment:
can we create a union or enum type for the token standards?

Reply:
Created a enum!